exercises/book-store/description.md

```diff
@@ -1,5 +1,3 @@
-To try and encourage more sales of different books from a popular 5 book series, a bookshop has decided to offer discounts of multiple-book purchases.
-
```

Comment:
@ErikSchierboom I have been following the discussion here and I agree with you on this point but I would like to add that the hyphen should be removed if `multiple` is left in.  I like `multiple book` better than `multi-book`.  I never really noticed it before till the topic of consolidation came up.